components/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/..)
 
 # Set global targets
 add_custom_target(genf90
-  DEPENDS ${CIMEROOT}/src/externals/genf90/genf90.pl)
+  DEPENDS ${CIMEROOT}/CIME/non_py/externals/genf90/genf90.pl)
```

Comment:
Is this where the dependence on CIME genf90 comes in to the E3SM build?  Lets just our own in E3SM/externals.

Reply:
@rljacob , yes, that's one of the places. CIME also uses it internally in the important Tools/Makefile. Since the problem is in a shared lib, it's the Makefile one that's the problem.